dataflow/snippets/write_pubsub.py

```diff
@@ -26,6 +26,8 @@
 
 
 def item_to_message(item: Dict[str, Any]) -> PubsubMessage:
+    from apache_beam.io import PubsubMessage
```

Comment:
This was already imported on line 21. Does it need to be imported again?

Reply:
Yes this fixes an issue where on Dataflow, this function runs on a worker VM and the session state isn't preserved. 

I added a code comment to explain the rationale, it's not super intuitive IMO.

There are some other workarounds, e.g. at runtime you can pass a flag to Dataflow to save the session state, but I think that's more error prone. 

More info here: https://cloud.google.com/dataflow/docs/guides/common-errors#name-error